src/Tools/dotnet-dsrouter/Program.cs

```diff
@@ -211,7 +211,7 @@ private static Option WebSocketURLAddressOption() =>
                 description: "The router WebSocket address using format ws://[host]:[port]/[path] or wss://[host]:[port]/[path]. " +
                                 "Launch app with WasmExtraConfig property specifying diagnostic_options with a server connect_url")
             {
-                Argument = new Argument<string>(name: "webSocketURI", getDefaultValue: () => "")
+                Argument = new Argument<string>(name: "webSocketURI", getDefaultValue: () => "ws://127.0.0.1:8088/diagnostics")
```

Comment:
This seems to already be set as the default value as part of WeSocketServerRouterFactory, https://github.com/dotnet/diagnostics/blob/0d9cb0b715621695b84c473d04dc29d9f8f40bd3/src/Microsoft.Diagnostics.NETCore.Client/DiagnosticsServerRouter/DiagnosticsServerRouterFactory.cs#L359

Reply:
Yes, but it also needs to flow into `WebSocketServerImpl.ParseWebSocketURL` otherwise it throws on empty value and continues starting silently without actually working!

Reply:
ok, let me check that, sound like that parsing should be done elsewhere.